PROJECT

```diff
@@ -2,7 +2,7 @@ NAME = 'SoLoader'
 
 GROUP = 'com.facebook.soloader'
 ARTIFACT = 'soloader'
-VERSION='0.6.1'
+VERSION='0.6.2'
```

Comment:
can you revert the version bump? We already have 0.7.0 released

Reply:
I have reset the change of version bump. :)